mudz.Common/EqualityComparisonHelpers.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace mudz.Common
+{
+	public static class EqualityComparisonHelpers
```

Comment:
Annoyingly, very few `Linq` constructs allow for use of `IEqualityComparer<T>`, so I made a quick helper.